wazo_calld/plugins/voicemails/storage.py

```diff
@@ -219,6 +219,9 @@ def parse(self, fobj):
         result = {}
         parsed = set()
         for line in fobj:
+            line = '='.join([x.strip() for x in line.decode('utf-8').split('=')])
```

Comment:
Quick question. Would it work without decoding and encoding?
```python3
line = b'='.join([x.strip() for x in line.split(b'=')])
line = line.lstrip()
```

Reply:
I also think `line = line.lstrip()` is redundant since `x.strip()` already stripped the first element of the line. This line could probably be removed.

Reply:
Yes you are right, I removed the encoding and the ```lfstrip``` line :)